crates/compiler/builtins/roc/Set.roc

```diff
@@ -72,6 +100,18 @@ expect
 
     expected == actual
 
+## Counts the number of values in a given `Set`.
+## ```
+## numbers =
+##     Set.empty {}
+##     |> Set.insert "One"
```

Comment:
Can we make these not strings. It just feels really weird using strings here.

Reply:
Updated. Thank you